pcdsdevices/epics_motor.py

```diff
@@ -177,17 +177,27 @@ def format_status_info(self, status_info):
 """
 
     @property
-    def limits(self):
+    def limits(self) -> tuple[float, float]:
         """Override the limits attribute"""
         return self._get_epics_limits()
 
-    def _get_epics_limits(self):
+    def _get_epics_limits(self) -> tuple[float, float]:
         limits = self.user_setpoint.limits
         if limits is None or limits == (None, None):
             # Not initialized
             return (0, 0)
         return limits
 
+    @limits.setter
+    def limits(self, lims: tuple[float, float]):
+        """
+        Write to the epics limits on setattr
+
+        Expects a tuple of two elements, the low and high limits to set.
+        """
+        self.low_limit_travel.put(min(lims))
```

Comment:
Maybe `if not len(lims) == 2: raise ValueError(...)`

And _maybe_ have special handling for `if min(lims) == max(lims)` ? That implies "disable limits or low, high = (0, 0)" in the motor record world I believe

Reply:
Both of these are worth doing